File: Trailing0.c
#include <stdio.h>

int main() {
    int n, count = 0;

    printf("Enter a number: ");
    scanf("%d", &n);

    for(int i = 5; n / i >= 1; i *= 5) {
        count += n / i;
    }

    printf("Trailing zeros in %d! = %d", n, count);

    return 0;
}

File: Trailing0.java
import java.util.Scanner;

public class TrailingZeros {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int n = sc.nextInt();

        int count = 0;

        for(int i = 5; n / i >= 1; i *= 5) {
            count += n / i;
        }

        System.out.println("Trailing zeros in " + n + "! = " + count);

        sc.close();
    }
}
